_includes/related_posts.html

```diff
@@ -51,6 +58,13 @@
 					{% assign author = relevantPostAttributes[5] %}
 					{% assign postContent = relevantPostAttributes[6] %}
 					{% assign postDate = relevantPostAttributes[7] %}
+					{% assign abstract = relevantPostAttributes[8] %}
+					
+					{% if abstract %}
```

Comment:
@alvarogarcia7 and @Gryff, the change I made in that file is at line 5, so git should be able to merge those two without issues.. right?